notebooks/agent_powered_retrieval.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p><code>&nbsp;&nbsp;&nbsp;&nbsp;if len(retrieved_documents.get("documents")) &gt; top_k:</code></p><p> </p><p>can be removed. </p><p> </p><p><code>retrieved_documents["documents"] = retrieved_documents["documents"][:top_k]</code></p><p>should always work if <code>top_k</code> is an integer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>